src/test/ui/exhaustive_integer_patterns.stderr

```diff
@@ -0,0 +1,39 @@
+error: unreachable pattern
+  --> $DIR/exhaustive_integer_patterns.rs:32:9
+   |
+LL |         200 => {} //~ ERROR unreachable pattern
+   |         ^^^
+   |
+note: lint level defined here
+  --> $DIR/exhaustive_integer_patterns.rs:13:9
+   |
+LL | #![deny(unreachable_patterns)]
+   |         ^^^^^^^^^^^^^^^^^^^^
+
+error[E0004]: non-exhaustive patterns: `128u8...255u8` not covered
+  --> $DIR/exhaustive_integer_patterns.rs:37:11
+   |
+LL |     match x { //~ ERROR non-exhaustive patterns: `128u8...255u8` not covered
+   |           ^ pattern `128u8...255u8` not covered
+
+error[E0004]: non-exhaustive patterns: `11u8...20u8`, `30u8...35u8`, `35u8...70u8` and 1 more not covered
+  --> $DIR/exhaustive_integer_patterns.rs:42:11
+   |
+LL |     match x { //~ ERROR non-exhaustive patterns
+   |           ^ patterns `11u8...20u8`, `30u8...35u8`, `35u8...70u8` and 1 more not covered
+
+error: unreachable pattern
+  --> $DIR/exhaustive_integer_patterns.rs:53:9
+   |
+LL |         -2..=20 => {} //~ ERROR unreachable pattern
+   |         ^^^^^^^
+
+error[E0004]: non-exhaustive patterns: `-128i8...-5i8`, `120i8...121i8` and `121i8...127i8` not covered
+  --> $DIR/exhaustive_integer_patterns.rs:50:11
+   |
+LL |     match x { //~ ERROR non-exhaustive patterns
+   |           ^ patterns `-128i8...-5i8`, `120i8...121i8` and `121i8...127i8` not covered
```

Comment:
Could adjacent ranges be merged, or at least not overlap?  `121` is in two of the uncovered ranges, and it would be nice if the message was "patterns `-128i8...-5i8` and `120i8...127i8` not covered" instead.